test/runtests.jl

```diff
@@ -54,11 +54,11 @@ include("test_utils.jl")
     println(" ")
     @info "Ran expected_loglik tests"
 
-    include("sparse_variational.jl")
+    include("SparseVariationalApproximationModule.jl")
     println(" ")
-    @info "Ran svgp tests"
+    @info "Ran sva tests"
```

Comment:
Maybe avoid using abbreviation like sva?

Reply:
I'll add the change to a future PR